serverless.yml

```diff
@@ -4,7 +4,7 @@ custom:
   stage: ${opt:stage, 'staging'}
   pythonRequirements:
     dockerizePip: true
-    slim: true
+    slim: false # jsonschema package fails with true: https://github.com/Julian/jsonschema/issues/628#issuecomment-564456058
```

Comment:
in case we add more dependencies and need the slimming, I think another solution is these pythonRequirements:
```
    slim: true
    slimPatternsAppendDefaults: false
    slimPatterns:
      # These are the default slim patterns, minus *.dist-info.
      - "**/*.py[c|o]"
      - "**/__pycache__*"
```